Dockerfile

```diff
@@ -145,6 +145,10 @@ RUN pip3 install git+https://github.com/cov-lineages/constellations.git@v0.1.1
 RUN pip3 install git+https://github.com/cov-lineages/pango-designation.git@19d9a537b9
 RUN pip3 install pysam==0.19.1
 
+# Add evofr for forecasting
+# Not available on PyPI, so install via git URLs
+# Currently not pinning to any commit yet since it's still in active development
+RUN pip3 install git+https://github.com/blab/evofr.git
```

Comment:
This should come after `ARG CACHE_DATE` otherwise an older version will stay in the cache indefinitely.

Currently, only tools under the "Nextstrain components" section come after `CACHE_DATE`, since we want to have the latest version of all Nextstrain components (either latest release or latest on default branch) without the hassle of updating pins. I'm not familiar with `evofr` but if it's part of the Nextstrain toolkit as @trvrb describes, then it would make sense to put it in that section anyways. Otherwise the section could be renamed to "Nextstrain components and unpinned programs".

Reply:
Ah, thank you! I had no idea how Docker build caches work.

Reply:
After a good month of working on this repo, I understand it more now but still not 100%!